backend/routers/plugins.py

```
@@ -109,3 +116,129 @@ def get_plugin_money_made(plugin_id: str):
+    if plugin.works_externally() and plugin.external_integration.setup_completed_url:
+        res = requests.get(plugin.external_integration.setup_completed_url + f'?uid={uid}')
+        print('enable_plugin_endpoint', res.status_code, res.content)
+        if res.status_code != 200 or not res.json().get('is_setup_completed', False):
+            raise HTTPException(status_code=400, detail='Plugin setup is not completed')
+    increase_plugin_installs_count(plugin_id)
+    enable_plugin(uid, plugin_id)
+    return {'status': 'ok'}
+
+
+@router.post('/v2/plugins/disable')
+def disable_plugin_endpoint(plugin_id: str, uid: str = Depends(auth.get_current_user_uid)):
```

Ruff

Redefinition of unused `disable_plugin_endpoint` from line 40

(F811)